src/doc.d

```diff
@@ -820,11 +820,14 @@ extern (C++) void emitComment(Dsymbol s, OutBuffer* buf, Scope* sc)
                 for (size_t i = 0; i < dc.a.dim; i++)
                 {
                     Dsymbol sx = dc.a[i];
+                    // the added linebreaks in here make looking at multiple
+                    // signatures more appealing
                     if (i == 0)
                     {
                         size_t o = buf.offset;
                         toDocBuffer(sx, buf, sc);
                         highlightCode(sc, sx, buf, o);
+                        buf.writestring("$(BR)");
```

Comment:
Maybe something like `$(DDOC_OVERLOAD_SEPARATOR)` instead of a hard-coded linebreak? I think a configurable mechanism would be preferable.